### ISSUE TITLE
JSyphon server lazy initialised leads to unexpected behaviour

### ISSUE THREAD

**Issue by cansik** [Syphon/Processing#34]
As mentioned in the PR #33, following code should be considered as valid usage of the API (nothing prevents me as developer of doing it):

```java
void sendImageIfNecessary(PImage img) {
    if(syphon.hasClients())
        syphon.sendImage(img);
}
```

But this is not working, because the JSyphon sever is lazy initialised in the `sendImage` method. With the PR applied, syphon will never be initialised, because the server will never be initialised because `sendImage` will never be called.

Why is the server lazy initialised and why is there no method to explicitly start the server? I think this would make a lot of sense, not only architecture-wise, but also performance-wise (initialise it in the beginning and not when the frame-sending should happen).

In the implementation there is following comment:
```java
// The server needs to be created after setup and all the 
// JOGL initialization has taken place. Otherwise frame
// sending doesn't work...
```

This makes totally sense, but it still would make sense to have the control, when the server is initialised.

**Comment by cansik** [comment]
Just to make clear that the above code makes sense, I have performed some profiling on a processing sketch with syphon:

Check if `hasClients` before send, without clients attached:
![image](https://user-images.githubusercontent.com/5220162/35771023-8f9cdb98-0926-11e8-942a-376e8d6f9b7b.png)

Check if `hasClients` before send, with clients attached:
![image](https://user-images.githubusercontent.com/5220162/35771038-b3a6eb64-0926-11e8-8fe6-8848aa5e3bb6.png)

Default `sendImage` without clients attached (current API behaviour):
![image](https://user-images.githubusercontent.com/5220162/35771054-edb27300-0926-11e8-915c-e841d1379ac1.png)

As you can see in the last picture, the performance is used for  `publishFrameTexture` even if no client is attached to the server.

**Comment by bangnoise** [comment]
fixed in b3f78c5dba30c53f5fb36aa391458ccc79fa5793
